bin/postgres-ha/sql/post-existing-init.sql

```diff
@@ -13,5 +13,16 @@
  * limitations under the License.
  */
 
-CREATE USER ${PATRONI_SUPERUSER_USERNAME} WITH SUPERUSER PASSWORD $$${PATRONI_SUPERUSER_PASSWORD}$$;
-CREATE USER ${PATRONI_REPLICATION_USERNAME} WITH REPLICATION PASSWORD $$${PATRONI_REPLICATION_PASSWORD}$$;
+/**
+ * Check that there a default user, and if not, then create create it.  If the user does exist
+ * then update its password.
+ */
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = '${PGHA_USER}') THEN
+        CREATE ROLE "${PGHA_USER}" WITH LOGIN PASSWORD '${PGHA_USER_PASSWORD}';
```

Comment:
Please emulate the `$$` used for strings above. You can wrap the anonymous block with something like DO $block$ so then `$$` still works for the password strings.

Reply:
Cool, will do!

Reply:
So after fighting with this, I just ended up reverting back to just running the `CREATE` and `ALTER` statements directly (i.e. no longer inside of a function).  The reason for this was that `envsubst` was seeing `$block$` and thinking `$block` is an env var, causing an invalid script.  And since there don't appear to be any chars that can be used when defining the string inside of `$block$` to prevent this from happening, I was not able to find a way around this.

@cbandy and I did discuss some potential ways to avoid these issues in the future, e.g. using things like `psql` variables.  However, since currently the `envsubst` approach is used for `setup.sql` in this same code as well, it seemed like the best answer for now was to keep both consistent and user expectations the same around the behavior of these scripts (and this is essentially how this script always worked).  And just seeing a "role already exists" error in the event the user defined for the `CREATE` already exists (which is now the only role this script creates) seem innocuous enough.

If any questions, just let me know.